examples/rstudio/test.bats

```diff
@@ -0,0 +1,32 @@
+true
+# shellcheck disable=SC2034
+CH_TEST_TAG=$ch_test_tag
+
+load "${CHTEST_DIR}/common.bash"
+
+setup () {
+    scope full
+    prerequisites_ok "rstudio"
+}
+
+@test "${ch_tag}/test rstudio-server" {
```

Comment:
Looks like this just tests whether it's possible to log into Rstudio? Can you actually compute something? In either case, a comment explaining what's being tested would be helpful.

Reply:
Yes. The test just checks if we can login in to rstudio. My initial plan was to use selenium to perform a computation in rstudio after logging in by putting some code into the code editor then clicking the run button. The way rstudio is written made it basically impossible to get the frame that the code editor was in with selenium. It appears they just used a transpiler from java to javascript which renders everything a mess. I figured that since rstudio uses the local R installation as a backend testing that installation and that a user could access rstudio itself is sufficient. I'll add a comment.

Reply:
I don't see a comment?